src/aat/resources/features/F-032/F-032_Base_Test_Data.td.json

```diff
@@ -0,0 +1,27 @@
+{
+  "_guid_": "F-032_Base_Test_Data",
+
+  "user": {
+    "_extends_": "Common_User_For_Request"
+  },
+
+  "productName": "CCD Data Store",
+  "operationName": "get printable documents",
+  "method": "POST",
+  "uri": "/callback/jurisdictions/{jid}/case-types/{ctid}/documents",
+
+  "request": {
+    "_extends_": "Common_Request",
+    "pathVariables": {
+      "jid": "AUTOTEST1",
+      "ctid": "AAT_AUTH_15"
+    },
+    "body": {
+      "id": 1575901743247852
```

Comment:
Why is it hardcoded in the base file, Shouldn't it be dynamic here or S-124?

Reply:
was waiting for the DSL changes as there was no way to give dynamic data to request.body.id - it uses the dynamic case id now